### DISCUSSION TITLE
Problemi dopo ripristino backup

### DISCUSSION THREAD

**Discussion by fmtoken** [indomus/forum#2372] [HUB: Home Assistant]
Buonasera, dopo aver ripristinato l'intero sistema con un backup completo non vedo più i sensori zigbee.
Come broker ho mosquitto, come BRIDGE/Gateway zigbee2mqtt e come coordinator il SONOFF ZBDongle-P. C'è modo di recuperarli oppure devo collegare nuovamente ogni sensore?
Grazie

**Comment by snakuzzo** [comment]
In teoria, se nel backup che hai fatto c'era anche la directory data di zigbee2mqtt, dopo il restore (sempre se hai fatto tutto nel modo corretto) avresti dovuto "vedere" tutto quello che vedevi prima.

**Reply by fmtoken** [reply]
Grazie per la risposta... effettivamente ho fatto un ripristino anomalo in quanto se facevo il ripristino dal backup completo nella prima schermata che si presenta quando si fa una nuova installazione, al termine, l'utilizzo della CPU aumentava notevolmente senza più riscendere e il Raspberry diventava inutilizzabile (lo stesso motivo per cui ho deciso di ripristinare il sistema quindi immagino che anche i backup, ne ho provati diversi, siano corrotti). Ho quindi provato a configurare il sistema come se fosse nuovo installando i vari componenti aggiuntivi e caricando successivamente i backup dei componenti aggiuntivi. Adesso funziona tutto regolarmente salvo il fatto che non vedo i sensori zigbee che avevo. C'è modo di estrapolare dal backup solo i file necessari per il ripristino dei sensori? Grazie

**Reply by snakuzzo** [reply]
te l'ho detto sopra...il backup di zigbee2mqtt è la dir `data` di zigbee2mqtt. Spegni zigbee2mqtt, ripristina quella directory e riavvia zigbee2mqtt. Se non va ti tocca rifare il pairing dei device
